--- tmux21.rb.orig	2016-07-08 15:43:47.000000000 +0900
+++ tmux21.rb	2016-07-08 16:05:35.000000000 +0900
@@ -1,4 +1,4 @@
-class Tmux < Formula
+class Tmux21 < Formula
   desc "Terminal multiplexer"
   homepage "https://tmux.github.io/"
 
@@ -21,14 +21,6 @@
     end
   end
 
-  bottle do
-    cellar :any
-    revision 2
-    sha256 "815920cd38a8102360f7d667271d9c724f41087dd79be433db29259390ef8011" => :el_capitan
-    sha256 "93e2156c3c7e1c9f3f4b86dd84617e7519e9bee630f1e8769e00a91aa341d274" => :yosemite
-    sha256 "03c4ca001f72a1623393c0ec9406dfd82b7e449d745762a6e761da6a95d0fbd9" => :mavericks
-  end
-
   head do
     url "https://github.com/tmux/tmux.git"
 
@@ -67,4 +59,11 @@
   test do
     system "#{bin}/tmux", "-V"
   end
+
+  def patches
+    [
+      "https://gist.githubusercontent.com/waltarix/1399751/raw/8c5f0018c901f151d39680ef85de6d22649b687a/tmux-ambiguous-width-cjk.patch",
+      "https://gist.githubusercontent.com/waltarix/1399751/raw/dc11f40266d9371e730eff41c64a70c84d34484a/tmux-pane-border-ascii.patch"
+    ]
+  end
 end
